AUTHORS.md

```diff
@@ -30,3 +30,4 @@ Other contributors:
  - [Dario Ernst](https://github.com/Nebukadneza) - [Contributions](https://github.com/Mailu/Mailu/commits?author=Nebukadneza)
  - [Florent Daigniere](https://github.com/nextgens) - [Contributions](https://github.com/Mailu/Mailu/commits?author=nextgens)
  - [Alexander Graf](https://github.com/ghostwheel42) - [Contributions](https://github.com/Mailu/Mailu/commits?author=ghostwheel42)
+ - [Sebastian Wilke](https://github.com/Encotric) - [Contributions](https://github.com/heviat/Mailu/commits?author=encotric)
```

Comment:
Please remove this. Every author who wants to be listed here, must send a PR for this by himself. This ensures the person has given consent to be listed here.

Reply:
Makes sense, done